source/user-manual/agent/agent-management/remote-upgrading/custom-repository.rst

```diff
@@ -11,19 +11,19 @@ WPK files must be named using the following pattern:
 
 .. code-block:: none
 
-    wazuh_agent_W_X_Y_Z.wpk
+    wazuh_agent_W_X_Y.Z.wpk
 
 Where:
```

Comment:
We can create lists without indentation. We use three-spaces tabulation. For example `-  `

```
Where:

-  ``W`` represents the release version.
-  ``X`` denotes the target operating system.
-  ``Y`` indicates the machine architecture. Empty for Windows.
-  ``Z`` represents the package type included in the WPK file. Empty for Windows.
```